deployment/ansible/conf/jophiel.yml.j2

```diff
@@ -105,3 +105,9 @@ jophiel:
       subject: UNUSED
       body: UNUSED
 {% endif %}
+
+  superadminCreator:
+    enabled: true
+    username: superadmin
+    initialPassword: {{ jophiel_superadmin_initial_password }}
```

Comment:
```suggestion
    initialPassword: {{ jophiel_superadmin_initialPassword }}
```

Reply:
OK

Reply:
Changed to `jophiel_superadminCreator_initialPassword` for consistency with actual config key